Online/RainMeadowModManager.cs

```diff
@@ -8,23 +8,48 @@ namespace RainMeadow
     public class RainMeadowModManager
     {
         public static string[] GetActiveMods()
+        { 
+            return ModManager.ActiveMods
+                .Where(mod => Directory.Exists(Path.Combine(mod.path, "modify", "world"))
+                    || highImpactMods.Contains(mod.id)
+                    || cheatMods.Contains(mod.id))
+                .Select(mod => mod.id)
+                .ToArray();
+        }
+        
+        public static readonly string[] highImpactMods = {
+            "rwremix",
+            "moreslugcats",
+            "keepthatawayfromme",  // needs extra syncing to work
+        };
+
+        public static readonly string[] cheatMods = {
+            "devtools",
+            "maxi-mol.mousedrag",
+            "fyre.BeastMaster",
+            "slime-cubed.devconsole",
+            "zrydnoob.UnityExplorer",
+            "warp",
+            "pushtomeow",  //to make them mad >:)
+            "presstopup",
+            "no-damage-rng",
```

Comment:
consider moving this to high impact instead

Reply:
Which one?